lord/time_lord_test.go

```diff
@@ -306,6 +373,26 @@ var _ = DescribeTable("A range with a location and a previous time", (testCase).
 		nowDay:  time.Wednesday,
 
 		result: false,
+		latest: &expectedTime{hour: 13},
+		list: []expectedTime{
+			{hour: 13, weekday: time.Wednesday},
```

Comment:
Could you explain a bit why in here it doesn't return two times etc. 13:00 and 13:01 ? 13:01 is in the same day and falls in the range. Thx!

Reply:
This particular test case is exactly what @vito raised in [this question](I'm having a hard time interpreting what start + stop without interval should even mean).

The de facto requirement (i.e. the existing implementation) is that exactly 1 version is generated when a `start`/`stop` window is defined _**without**_ an interval.

Since the previous version of `13:00` falls within the `start`/`stop` window and there is no interval defined, `check` returns false to indicate that now (`13:01`) is _**not**_ a valid version.
    NOTE: the logic for `check` and the result of this test case are unchanged

More generally speaking, any test case where the `result` from `check` is `false` should _**not**_ contain `now` in the `list` of expected versions.

Reply:
Got it. Could we include this implication in the documentation of `start/stop` i.e.

> exactly 1 version is generated when a start/stop window is defined without an interval